src/components/search-bar/style.scss

```diff
@@ -20,11 +20,19 @@
     }
 
     .popup {
-        display: none;
+        max-height: 0px;
+        transition: max-height .25s ease-in-out, padding .25s ease-in-out;        
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
+        padding-top: 0px;
+        padding-bottom: 0px;
+
 
         &.show {
-            display: flex;
-            flex-direction: column;
+            max-height: 100vh;
```

Comment:
Since the `max-height` is `100vh` the animation looks a bit strange, I would change the transition to be linear or add a more advanced solution for setting the max height